src/stores/user.ts

```diff
@@ -1,15 +1,26 @@
 import { defineStore } from "pinia";
 import { useStorage } from "@vueuse/core";
+import { useCurrentUser, Roles } from "@/features/user";
 
 export const useUserStore = defineStore("user", {
   state: () => {
     return {
       preferences: useStorage("preferences", { kanaal: "" }),
+      userData: useCurrentUser(),
     };
   },
   actions: {
     setKanaal(kanaal) {
       this.preferences.kanaal = kanaal;
     },
   },
+  getters: {
+    hasAdminPostRole(state) {
```

Comment:
zou een rol kissredactie verwachten of kissberichtenredacteur. adminpost zegt me niets. naam even afstemmen via sytske?

Reply:
ga ik doen

Reply:
in overleg de rol check eruit gehaald